dds_web/api/user.py

```diff
@@ -232,6 +195,76 @@ def add_user_to_project(existing_user, project, role):
             ),
         }
 
+    @staticmethod
+    @logging_bind_request
+    def compose_and_send_email_to_user(userobj, mail_type, link=None, project=None):
+
+        # Compose and send email
+        unit_name = None
+        project_id = None
+        deadline = None
+        if auth.current_user().role in ["Unit Admin", "Unit Personnel"]:
+            unit = auth.current_user().unit
+            unit_name = unit.external_display_name
+            unit_email = unit.contact_email
+            sender_name = auth.current_user().name
+            subject_subject = unit_name
```

Comment:
Doing this should only be possible for Unit Admin and Unit Personnel roles right? Or Super Admin later on I guess.

Reply:
This was already in the code for the invite, I haven't changed that bit.

Reply:
We can leave this for now, I need to check something.